### DISCUSSION TITLE
Is it legitimate to RunUMAP on PCA after integration?

### DISCUSSION THREAD

**Discussion by weishwu** [satijalab/seurat#9434] [General]
Sorry if this is a silly question. After running integration using, for example:
```
data_integrated = IntegrateLayers(
  object = data_merged,
  method = RPCAIntegration,
  normalization.method = "SCT",
  new.reduction = "integrated.rpca")
```

According to the vignette, clustering and UMAP can be generated using the integrated reduction:
```
data_integrated <- data_integrated %>% FindNeighbors(reduction = "integrated.rpca", dims = 1:30) %>% FindClusters(resolution = 0.3) %>% RunUMAP(dims = 1:30, reduction = "integrated.rpca")
```

I wonder if it's legitimate to use PCA for UMAP, even though `FindNeighbors` is still run using "integrated.rpca":
```
data_integrated <- data_integrated %>% FindNeighbors(reduction = "integrated.rpca", dims = 1:30) %>% FindClusters(resolution = 0.3) %>% RunUMAP(dims = 1:30, reduction = "pca")
```

In the data I'm analyzing, the PCA-based UMAP appears to be more neat than the integrated-reduction-based UMAP.

**Comment by samuel-marsh** [comment]
Hi,

Not member of dev team but hopefully can be helpful.  Two things to note.  First, I'm going to move this issue and your other recently posted issues to the discussion section to hopefully get input from broader community as they are not so much issues with the package but questions on analysis practices.  

Second, on the topic of this question, if you are doing integration you want to run things using the integrated pipeline, especially if that's what you are using for other aspects of analysis.  However, I would also note that you should only use integration in an analysis if it is necessary, it should not be the default.  If the analysis works without using integration then there isn't a need to use it.  This just requires iterating the analysis and examining the results to determine the best pipeline for an individual dataset.

Best,
Sam
